news/10703.feature.rst

```diff
@@ -0,0 +1 @@
+Start utilising ``rich`` for presenting error messages in a consistent format.
```

Comment:
Rich itself uses normal font and standard capitalisation rules for a proper noun, for example: https://rich.readthedocs.io/en/latest/introduction.html

And prefer plain English "using" rather than "utilising".  

```suggestion
Start using Rich for presenting error messages in a consistent format.
```